src/actions/Card.js

```diff
@@ -18,6 +18,28 @@ export const getCards = async () => {
     });
 };
 
+export const getCardsByIds = async (ids) => {
+  return fetch(urls.api.card.getIds, {
+    method: "POST",
```

Comment:
A "get" request should use the `GET` method. I'd recommend [this](https://stackoverflow.com/questions/35038857/setting-query-string-using-fetch-get-request) for building the URL a bit easier. You might have to change the request handler in `pages/api/card/getIds` a little bit.

Reply:
Meant to rename this getIds to be a POST method, the number of ids that should be accepted by the endpoint exceeds the data limit of a GET request.